app/routes/index.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+
+
```

Comment:
Also I'm guessing you want the model hook that returns all reminders to be in routes reminders